toolset/setup/linux/prerequisites.sh

```diff
@@ -3,12 +3,7 @@
 set -x
 export DEBIAN_FRONTEND=noninteractive
 
-source $FWROOT/toolset/setup/linux/bash_functions.sh
-
-RETCODE=$(fw_exists fwbm_prereqs_installed)
-[ ! "$RETCODE" == 0 ] || { \
-  echo "Prerequisites installed!"; 
-  return 0; }
+source ./toolset/setup/linux/bash_functions.sh
```

Comment:
Is this necessary? I don't think you're actually using any of the functions in here, are you?